mrtarget/plugins/gene/tractability.py

```diff
@@ -86,9 +87,9 @@ def build_json(self, filename=Config.TRACTABILITY_FILENAME):
                         'predicted_tractable': float(Predicted_Tractable)
                     },
                     'top_category': Category,
-                    'ensemble': ensemble,
-                    'high_quality_compounds': High_Quality_ChEMBL_compounds,
-                    'small_molecule_genome_member': Small_Molecule_Druggable_Genome_Member
+                    'ensemble': float(ensemble), # drugebility score not used at the moment but in a future
```

Comment:
I think it would be better if these conversions were handled more explicitly and less inline - with appropriately granular error messages and recovery in case of problems.